ecosystem/sep-0036.md

```diff
@@ -0,0 +1,137 @@
+## Preamble
+
+```
+SEP: 0036
+Title: Regulated Assets Transacting
+Author: Howard Chen <@howardtw>
+Status: Draft
+Created: 2020-10-26
+Version 0.1.0
+```
+
+## Simple Summary
+
+Regulated Assets provide ecosystem support for assets that require an issuer’s approval (or a delegated third party’s approval, such as a licensed securities exchange) on a per-transaction basis. It standardizes the identification of such assets as well as defines the protocol for performing compliance checks and requesting issuer approval.
+
+This SEP is based on [SEP-0008](sep-0008.md), but the issuer no long has to a co-signer on the account holding the regulated asset. It also updates the API of the approval server to better align with other active SEPs.
```

Comment:
I think you should drop this sentence. You can mention this in the PR and/or commit comment and beyond that this SEP should standalone.